data_override/README.MD

```diff
@@ -7,29 +7,29 @@
 - [How to use it?](README.MD#2-how-to-use-it)
 - [Converting legacy data_table to data_table.yaml](README.MD#3-converting-legacy-data_table-to-data_tableyaml)
 - [Examples](README.MD#4-examples)
+- [External Weight File Structure](README.MD#5-external-weight-file-structure)
 
 #### 1. YAML Data Table format:
 Each entry in the data_table has the following key values:
-- **gridname:** Name of the grid to interpolate the data to. The acceptable values are "ICE", "OCN", "ATM", and "LND"
-- **fieldname_code:** Name of the field as it is in the code to interpolate.
-- **fieldname_file:** Name of the field as it is writen in the file. **Required** only if overriding from a file
-- **file_name:** Name of the file where the variable is located, including the directory. **Required** only if overriding from a file
-- **interpol_method:** Method used to interpolate the field. The acceptable values are "bilinear", "bicubic", and "none". "none" implies that the field in the file is already in the model grid. The LIMA format is no longer supported. **Required** only if overriding from a file
+- **grid_name:** Name of the grid to interpolate the data to. The acceptable values are "ICE", "OCN", "ATM", and "LND"
+- **fieldname_in_model:** Name of the field as it is in the code to interpolate.
+- **override_file:** The parent key containing the nested keys related to overriding data from file. **Required** only if overriding from a file
```

Comment:
Concerned the use of parent and nested keys here could be confused with the idea of parent and nested grids. Is the standardized YAML terminology to use parent and nest or can we define it as we want?